src/ElmFormat.hs

```diff
@@ -83,23 +86,26 @@ resolveFiles inputFiles =
                 return $ Left ls
 
             Right files ->
-                return $ Right $ concat files
+                return $ Right $ concat $ files
+
 
+type ElmFile
```

Comment:
Would you prefer a `newtype` instead? Or no type at all and just using the tuple everywhere?

Reply:
Yeah, I think either `newtype` or `data` with named record fields is preferable.

Reply:
Ok, I used a data with named record fields.